docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -145,7 +145,7 @@ This is generally the path you should use in your code.
 ## Accessing ScPCA module results
 
 As an analysis modules matures, the Data Lab will port the analysis to a separate repository called [`OpenScPCA-nf`](https://github.com/AlexsLemonade/OpenScPCA-nf).
-This repository holds a [Nextflow workflow](https://www.nextflow.io/) to reproducibly run and generate results from completed analysis modules.
+This repository [holds a Nextflow workflow](../../ensuring-repro/openscpca-nf/index.md) to reproducibly run and generate results from completed analysis modules.
```

Comment:
I'm not sure we need the link directly to the repo here? maybe just to the docs page about it? And then maybe leave the Nextflow link? Or remove it completely.

Reply:
I just removed the repo link on line 147